src/js/initializers/initGlobals.ts

```diff
@@ -6,15 +6,17 @@ import TabHistories from "../state/TabHistories"
 import {Store} from "../state/types"
 import {createMemoryHistory} from "history"
 import tabHistory from "app/router/tab-history"
+import {ipcRenderer} from "electron"
 
-export default function initGlobals(store: Store) {
+export default async function initGlobals(store: Store) {
   global.windowId = getUrlSearchParams().id
   global.windowName = getWindowName()
   global.feature = (name, status) => store.dispatch(Feature.set(name, status))
   global.tabHistories = new Histories(TabHistories.selectAll(store.getState()))
   global.windowHistory = createMemoryHistory()
   global.windowHistory.replace(getUrlSearchParams().href)
   global.navTo = (path) => store.dispatch(tabHistory.push(path))
+  global.mainArgs = await ipcRenderer.invoke("get-main-args")
```

Comment:
All the args that were passed to the main process get put into the renderer as a global called `mainArgs`.